scripts/generate_usb_vif/constants/dt_constnats.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
```

Comment:
Please fix the typo in this file name.

Reply:
Sorry didn't notice it, updated it in new revision. Thank you